pkgs/servers/etebase/default.nix

```
@@ -0,0 +1,57 @@
+{ lib, stdenv, python3, fetchFromGitHub }:
+
+let
+  py = python3.override {
+    packageOverrides = self: super: {
+      django = super.django_3;
+    };
+  };
+in
+  with py.pkgs;
+
+buildPythonPackage  rec {
```

if this is just needed for the command, please use `buildPythonApplication`
```suggestion
buildPythonApplication  rec {
```
This will also avoid the need to wrap it later

---

So, I played with this a bit and I got an error when switching to `buildPythonAppliaction`, but then I found `toPythonApplication` used in `all-package.nix` and it by using it I can keep `buildPythonPackage` but still remove the `wrapped.nix`.

---

I'm pretty sure I need a python package not app for the python env setup to work anyway:
```
  pythonEnv = pkgs.python3.withPackages (ps: with ps;
    [ etebase-server daphne ]);
```
So, this is probably the best solution.

---

We only use `buildPythonPackage` if the package has a library portion. If it has both we use `toPythonApplication`. And if it is only an application we use `buildPythonApplication`.

---

Well, `daphne` and any other ASGIs need access to the python files of `etebase-server`, which is why I need to have it as a library as well.